x/multisig/msg.go

```diff
@@ -0,0 +1,30 @@
+package multisig
+
+import "github.com/iov-one/weave"
+
+// Path fulfills weave.Msg interface to allow routing
+func (CreateContractMsg) Path() string {
+	return pathCreateContractMsg
+}
+
+// Validate enforces sigs and threshold boundaries
+func (c *CreateContractMsg) Validate() error {
+	if err := weave.Address(c.Address).Validate(); err != nil {
```

Comment:
I'm not really a fan of letting the CreateContractMsg define it's own address. 
I can explain more later.